ampeg.testing/src/test/java/SupportPage.java

```diff
@@ -0,0 +1,17 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SupportPage extends PageObject {
+
+	public SupportPage(WebDriver driver) {
+		super(driver);
+	}
+	
+	public ManualsPage clickCurrentManualsLink() {
+		var currentManualsLink = this.driver.findElement(By.cssSelector("a[href='https://ampeg.com/support/manuals/'"));
```

Comment:
What if you're not testing in a production environment? The url may be different.